### PR TITLE
{Doc} Update instructions on using airgapped clouds

### PR DESCRIPTION
**Pull request by fengzhou-msft**
**Description**  

Add a list of known unsupported features in airgapped clouds.

**Testing Guide**  

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Doc

File: doc/use_cli_in_airgapped_clouds.md [hunk 1]
```diff
@@ -3,8 +3,15 @@
 ## Introduction
 While Azure CLI is not fully supported in an airgapped cloud which has no public internet connectivity, most commands can run without issues if the underlying service is supported in that cloud.
 
+Here is a list of known CLI features that are not supported in airgapped clouds:
+* Manage extensions with extension names. For instance, you can not install the `azure-devops` extension with `az extension add --name azure-devops`. You need to provide a local path to a wheel file or a URL for downloading the wheel file that is accessible in the airgapped cloud with `--source` option.
+* `az upgrade` to update to the latest CLI version. You can follow the below install instructions to update CLI.
+* `az find`
```
Comment (qwordy) [inline doc/use_cli_in_airgapped_clouds.md:9]: Any description of `az find`?

File: doc/use_cli_in_airgapped_clouds.md [hunk 2]
```diff
@@ -3,8 +3,15 @@
 ## Introduction
 While Azure CLI is not fully supported in an airgapped cloud which has no public internet connectivity, most commands can run without issues if the underlying service is supported in that cloud.
 
+Here is a list of known CLI features that are not supported in airgapped clouds:
+* Manage extensions with extension names. For instance, you can not install the `azure-devops` extension with `az extension add --name azure-devops`. You need to provide a local path to a wheel file or a URL for downloading the wheel file that is accessible in the airgapped cloud with `--source` option.
+* `az upgrade` to update to the latest CLI version. You can follow the below install instructions to update CLI.
```
Comment (qwordy) [inline doc/use_cli_in_airgapped_clouds.md:8]: what's the different between upgrade and update? I suggest use same word.

**Review by yungezz — APPROVED** [review]

File: doc/use_cli_in_airgapped_clouds.md (hunk 2 above)
Comment (fengzhou-msft) [reply doc/use_cli_in_airgapped_clouds.md]: Quite similar. Some [website](https://rocketit.com/difference-between-update-upgrade/) says upgrade has bigger changes than update.

File: doc/use_cli_in_airgapped_clouds.md (hunk 1 above)
Comment (fengzhou-msft) [reply doc/use_cli_in_airgapped_clouds.md]: added.

**Review by qwordy — APPROVED** [review]

File: doc/use_cli_in_airgapped_clouds.md [hunk 3]
```diff
@@ -4,13 +4,13 @@
 While Azure CLI is not fully supported in an airgapped cloud which has no public internet connectivity, most commands can run without issues if the underlying service is supported in that cloud.
 
 Here is a list of known CLI features that are not supported in airgapped clouds:
-* Manage extensions with extension names. For instance, you can not install the `azure-devops` extension with `az extension add --name azure-devops`. You need to provide a local path to a wheel file or a URL for downloading the wheel file that is accessible in the airgapped cloud with `--source` option.
+* Manage extensions with extension names. For instance, you cannot install the `azure-devops` extension with `az extension add --name azure-devops`. You need to provide a local path to a wheel file or a URL for downloading the wheel file that is accessible in the airgapped cloud with `--source` option.
```
Comment (qwordy) [inline doc/use_cli_in_airgapped_clouds.md:7]: What's the difference between "can not" and "cannot"?

Comment (fengzhou-msft) [reply doc/use_cli_in_airgapped_clouds.md:7]: > Both cannot and can not are acceptable spellings, but the first is much more usual. You would use can not when the "not" forms part of another construction such as "not only." For example:
> 
> These green industries can not only create more jobs, but also promote sustainable development of the land.

Link: https://www.lexico.com/grammar/cannot-or-can-not

**Pull request merged**